### PR TITLE
chore: use errors.As API instead of type conversion

### PR DESCRIPTION
**Pull request by mrkagelui**

### CODE REVIEW

File: business/sdk/sqldb/sqldb.go [hunk 1]
```diff
@@ -10,12 +10,13 @@ import (
 	"strings"
 	"time"
 
-	"github.com/ardanlabs/service/foundation/logger"
-	"github.com/ardanlabs/service/foundation/tracer"
 	"github.com/jackc/pgx/v5/pgconn"
 	_ "github.com/jackc/pgx/v5/stdlib"
 	"github.com/jmoiron/sqlx"
 	"go.opentelemetry.io/otel/attribute"
+
```
Comment (ardan-bkennedy) [inline business/sdk/sqldb/sqldb.go:17]: I like the change except the imports. Just keep 2 sets. Stdlib and non-stdlib

if you are not in the CONTRIBUTORS file can you add yourself please

**Comment by ardan-bkennedy** [comment]
I pulled the changes in. THANKS!

**Comment by mrkagelui** [comment]
Yeah sorry was in a rush, didn't fix the auto formatting from my IDE

**Comment by ardan-bkennedy** [comment]
No worries. It was a nice change. I had other work to do on the code for next week, so I just made the change.

**Pull request closed without merging**